pkgs/desktops/gnome/extensions/extensionRenames.nix

```diff
@@ -26,4 +26,16 @@
 
   # That extension is broken because of https://github.com/NixOS/nixpkgs/issues/118612
   #"flypie@schneegans.github.com" = null;
+
+  # These are overrides for the manually packaged ones (some of which got deleted – as in: replaced with
+  # some automatic packaging –, but we still need the "old" name for backwards compatibility).
+  "appindicatorsupport@rgcjonas.gmail.com" = "appindicator";
+  "unredirect@vaina.lt" = "disable-unredirect";
+  "drawOnYourScreen@abakkk.framagit.org" = "draw-on-your-screen";
+  "EasyScreenCast@iacopodeenosee.gmail.com" = "easyScreenCast";
+  "gnome-fuzzy-app-search@gnome-shell-extensions.Czarlie.gitlab.com" = "fuzzy-app-search";
+  "paperwm@hedning:matrix.org" = "paperwm";
+  "timepp@zagortenay333" = "timepp";
+  "TopIcons@phocean.net" = "topicons-plus";
```

Comment:
And this one.

Reply:
Makes sense, added a comment to all of them.